generators/entity-client/templates/react/src/main/webapp/app/entities/entity.reducer.ts.ejs

```diff
@@ -260,6 +260,9 @@ export const deleteEntity: ICrudDeleteAction<I<%= entityReactName %>> = id => as
     type: ACTION_TYPES.DELETE_<%= entityActionName %>,
     payload: axios.delete(requestUrl)
   });
+  <%_ if (pagination === 'pagination' || pagination === 'pager' || pagination === 'infinite-scroll' || pagination !== 'no' && pagination !== 'infinite-scroll') { _%>
```

Comment:
Pager isn't supported anymore

Reply:
The condition is a bit weird, check this out

| _value_         | pagination | pager | infinite-scroll | !(no \|\| infinite-scroll) | result |
|-----------------|------------|-------|-----------------|----------------------------|------------|
| pagination      | true       | false | false           | true                       | true       |
| pager           | false      | true  | false           | true                       | true       |
| infinite-scroll | false      | false | true            | false                      | true       |
| no              | false      | false | false           | false                      | false      |
| falsy           | false      | false | false           | true                       | true       |

Two interesting things here:
  1. only `no` resolves to `false`
  2. `false` resolves to `true`

So I guess it could be simplified by `!!pagination && pagination !== 'no'`

Reply:
The condition is a bit weird, check this out

| _value_         | pagination | pager | infinite-scroll | !(no \|\| infinite-scroll) | result |
|-----------------|------------|-------|-----------------|----------------------------|------------|
| pagination      | true       | false | false           | true                       | true       |
| pager           | false      | true  | false           | true                       | true       |
| infinite-scroll | false      | false | true            | false                      | true       |
| no              | false      | false | false           | false                      | false      |
| falsy           | false      | false | false           | true                       | true       |

Two interesting things here:
  1. only `no` resolves to `false`
  2. `false` resolves to `true`

So I guess it could be simplified by `!!pagination && pagination !== 'no'`